client/packages/common/src/intl/locales/en/common.json

```diff
@@ -577,16 +599,87 @@
   "placeholder.search-by-name-or-code": "Search by name or code",
   "placeholder.search-by-notes": "Search by notes",
   "preferences": "Preferences",
+  "report.actual": "Actual",
+  "report.actual-stock": "Actual stock",
+  "report.amc-12-months": "AMC (12 months)",
+  "report.amc-24-months": "AMC (24 months",
+  "report.authorized-by": "Authorized-by",
+  "report.collected-by": "Collected by",
+  "report.comments": "Comments",
+  "report.confirm-date": "Confirm date",
+  "report.confirmed-date": "Confirmed date",
+  "report.consumption": "Consumption",
+  "report.cost-price": "Cost price",
+  "report.cost-price-per-pack": "Cost price (per pack)",
+  "report.counted-packs": "Counted packs",
+  "report.created-date": "Created date",
+  "report.entered-code": "Entered code",
+  "report.expected-usage": "Expected usage",
+  "report.expiring": "Expiring",
+  "report.expiring-12-months": "Expiring in 12 months",
+  "report.expiring-6-months": "Expiring in 6 months",
+  "report.expiring-in-days": "Expiring in (days)",
+  "report.expiry-date": "Expiry date",
+  "report.extension": "Extension",
+  "report.global-total": "Global total",
+  "report.in-stock": "In stock",
+  "report.invoice": "Invoice",
+  "report.invoice-type": "Invoice type",
+  "report.issued": "Issued",
+  "report.item-code": "Item code",
+  "report.item-name": "Item name",
+  "report.line": "Line",
+  "report.months-cover": "Months cover",
+  "report.mos": "MOS",
+  "report.net-decrease": "Net Decrease",
+  "report.net-increase": "Net Increase",
+  "report.net-variance": "Net Variance",
+  "report.no-consumption": "No consumption",
+  "report.original": "Original",
+  "report.our-ref": "Our ref",
+  "report.out-of-stock": "Out of stock",
+  "report.overstocked": "Overstocked",
+  "report.pack-quantity": "Pack quantity",
+  "report.pack-size": "Pack size",
+  "report.packing-slip": "Packing slip",
+  "report.page": "Page",
+  "report.printed-date": "Printed date",
+  "report.quan": "Quan",
+  "report.received-from": "Received from",
+  "report.requested-from": "Requested from",
+  "report.requested-qty": "Requested Qty",
+  "report.requisition-category-message": "Requisition Category: Category Value",
+  "report.sell-price": "Sell price",
+  "report.shipped-date": "Shipped date",
+  "report.snapshot-packs": "Snapshot packs",
+  "report.stat": "Stat",
+  "report.stock-at-risk": "Stock at risk",
+  "report.stock-on-hand": "Stock on hand",
+  "report.stock-on-order": "Stock on order",
+  "report.stock-take-variance-report": "Stock Take Variance Report",
+  "report.stocktake-number": "Stocktake number",
+  "report.stocktake-sheet": "Stocktake sheet",
+  "report.suggested-qty": "Suggested Qty",
+  "report.supplied-to": "Supplied to",
+  "report.target-qty": "Target Qty",
+  "report.their-ref": "Their ref",
+  "report.understocked": "Understocked",
+  "report.usage-2-months-prior": "Monthly usage (2 months ago",
+  "report.usage-last-month": "Monthly usage (last month",
```

Comment:
Do you reckon you could add the closing `)` on to these (there are a few, screenshot in issue) as part of this? 🙏

Reply:
Opps sorry, missed a few